scripts/runtests.sh

```diff
@@ -231,6 +231,50 @@ grep -q 'finish: subiquity/Install/install/postinstall/run_unattended_upgrades:
 
 # The OOBE doesn't exist in WSL < 20.04
 if [ "${RELEASE%.*}" -ge 20 ]; then
+    # Test TCP connectivity (system_setup only)
+    clean
+    port=50321
+    LANG=C.UTF-8 timeout --foreground 60 \
```

Comment:
If you do much more of this, consider converting to something more like what subiquity/tests/api/test_api.py does.  That file is overdue to restructure to split the tests a bit (and so is runtests.sh).  My first API test was similar, forking the server process and using `curl` to communicate back with it.